src/devices/yokis.ts

```diff
@@ -0,0 +1,2885 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* TODO : remove eslint-disable when dev is done */
+
+import {Zcl} from 'zigbee-herdsman';
+import {ClusterDefinition} from 'zigbee-herdsman/dist/zspec/zcl/definition/tstype';
+
+import {repInterval} from '../lib/constants';
+import * as exposes from '../lib/exposes';
+import {logger} from '../lib/logger';
+import {
+    binary,
+    commandsLevelCtrl,
+    commandsOnOff,
+    commandsWindowCovering,
+    deviceEndpoints,
+    enumLookup,
+    forcePowerSource,
+    identify,
+    light,
+    numeric,
+    onOff,
+    windowCovering,
+} from '../lib/modernExtend';
+import {DefinitionWithExtend, KeyValue, KeyValueAny, ModernExtend, OnEvent, Tz} from '../lib/types';
+import * as utils from '../lib/utils';
+
+// #region Constants
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+const manufacturerOptions = {manufacturerCode: Zcl.ManufacturerCode.YOKIS};
+const NS = 'zhc:yokis';
+
+// PowerFailureMode (manuSpecificYokisSubSystem)
+const powerFailureModeEnum: {[s: string]: number} = {
+    last_state: 0x00,
+    off: 0x01,
+    on: 0x02,
+    blink: 0x03,
+};
+
+// timeTypeEnum (manuSpecificYokisLightControl)
+const timeTypeEnum: {[s: string]: number} = {
+    Seconds: 0x00,
+    Minutes: 0x01,
+};
+
+// operatingModeEnum (manuSpecificYokisLightControl)
+const operatingModeEnum: {[s: string]: number} = {
+    Timer: 0x00,
```

Comment:
Please make these lower case (also for the other enums)

Reply:
Done - renamed all enum items to lower case.